resources/aws/errors.go

```diff
@@ -28,69 +28,69 @@ const (
 
 // NotFound errors.
 
-var notFoundError = errgo.New("not found")
+var notFoundError = microerror.New("not found")
 
 // IsNotFound asserts NotFoundError.
 func IsNotFound(err error) bool {
-	return errgo.Cause(err) == notFoundError
+	return microerror.Cause(err) == notFoundError
 }
 
-var tooManyResultsError = errgo.New("too many results")
+var tooManyResultsError = microerror.New("too many results")
 
 // IsTooManyResults asserts tooManyResultsError.
 func IsTooManyResults(err error) bool {
-	return errgo.Cause(err) == tooManyResultsError
+	return microerror.Cause(err) == tooManyResultsError
 }
 
 // Delete errors.
 
-var resourceDeleteError = errgo.New("couldn't delete resource, it lacks the necessary data (ID)")
+var resourceDeleteError = microerror.New("couldn't delete resource, it lacks the necessary data (ID)")
```

Comment:
The error handling in the AWS operator in general is not always consistent with our other microservices. Something for boyscouting separately.